### PR TITLE
Add workflow function to user

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: UserBundle/Document/Authorization.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+     * @var string $name
+     *
+     * @ODM\Field(type="string")
+     */
+    protected $name;
+
+    /**
+     * @var ArrayCollection $workflowFunctions
+     *
+     * @ODM\ReferenceMany(targetDocument="OpenOrchestra\WorkflowFunction\Model\WorkflowFunctionInterface")
```
Comment (nicolasThal) [inline UserBundle/Document/Authorization.php]: with this relation, the WorkflowBundle is a requirement for the UserBundle

File: UserBundle/Model/AuthorizationInterface.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace OpenOrchestra\UserBundle\Model;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use OpenOrchestra\WorkflowFunction\Model\WorkflowFunctionInterface;
```
Comment (nicolasThal) [inline UserBundle/Model/AuthorizationInterface.php]: Same here

**Comment by itkg-ngilain** [comment]
:+1:

File: UserBundle/Document/Authorization.php (hunk 1 above)
Comment (itkg-nanne) [inline UserBundle/Document/Authorization.php]: So what should we do? If we just stock strings, each update of workflow will break the authorizations. I add this requirement in composer.json?

**Pull request closed without merging**